### DISCUSSION TITLE
[Dev Tools] Docs for hardhat plugin are broken

### DISCUSSION THREAD

**Discussion by cygaar** [zkSync-Community-Hub/zksync-developers#845] [Dev Tooling] [answered]
### Environment

Mainnet

### Select the Dev Tool you are using

Hardhat Plugins

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

This link (https://docs.zksync.io/build/tooling/hardhat/plugins/hardhat-zksync-verify) no longer works, it would be great if this could be a redirect to the updated doc links as this is the first link that pops up on Google search SEO.

**Comment by alhenaai** [comment, marked as answer]
Thank you for bringing this to our attention. It seems the link you mentioned is indeed broken. For the most current and accurate documentation on using the Hardhat plugins with ZKsync, please refer to the following updated links:

- [Hardhat ZKsync Verify Plugin Guide](https://docs.zksync.io/zksync-era/tooling/hardhat/plugins/hardhat-zksync-verify)
- [Getting Started with Hardhat and ZKsync](https://docs.zksync.io/zksync-era/tooling/hardhat/guides/getting-started)

These links provide comprehensive guidance on setting up and using the Hardhat plugins for ZKsync. If you have further questions or need more detailed assistance, please visit the [ZKsync developers discussions on Github](https://github.com/zkSync-Community-Hub/zksync-developers/discussions).

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by JackHamer09** [comment]
Thanks for report! Seems like this redirect was missed during refactor. Will work on fixing that!

**Comment by JackHamer09** [comment]
Fixed in https://github.com/matter-labs/zksync-docs/pull/294
